examples/vision/shiftvit.py

```diff
@@ -840,6 +909,125 @@ def __call__(self, step):
 print(f"Top 1 test accuracy: {acc_top1*100:0.2f}%")
 print(f"Top 5 test accuracy: {acc_top5*100:0.2f}%")
 
+"""
+## Save trained model
+
+Since we created the model by Subclassing, we can't save the model in HDF5 format.
+
+It can be saved in TF SavedModel format only. In general, this is the recommended format for saving models as well.
+"""
+tf.keras.models.save_model(model, "./ShiftViT")
+
+"""
+## Model inference
```

Comment:
Added a section on Model inference as I felt it would make the example more end-to-end and usable.

Please let me know if this section needs any fixes or if it can be made better in any way.